### PR TITLE
Migrate "internal" content from developer.mm.com

### PR DESCRIPTION
**Pull request by justinegeffen**
WIP; migration in progress.

### CODE REVIEW

**Comment by jasonblais** [comment]
@justinegeffen what are next steps on this one, or would we be able to merge what we have now and work iteratively to migrate rest of the content?

**Comment by justinegeffen** [comment]
@jasonblais, I'd like to merge what we have and work from there. :) I think this is good to go.

**Comment by jasonblais** [comment]
@justinegeffen Thanks! Propose an engineer lead to approve the changes, just for visibility. Perhaps Carlos Panato?

**Comment by justinegeffen** [comment]
@cpanato, this is some legacy content that I'm moving over from developers.mm.com/internal to the Handbook. I suspect some might be very old - would you be open to a quick review to let me know if anything has been deprecated? Once this is merged I'll start adding issues for updates needed. Thank you!

**Comment by justinegeffen** [comment]
@cpanato, just a reminder on this. I don't know exactly how much of it is usable but if there is some content we can keep that'd be great. :)

File: operations/research-and-development/engineering/infrastructure/aws.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# AWS
```
Comment (cpanato) [inline operations/research-and-development/engineering/infrastructure/aws.md]: this we can wipe out

File: operations/research-and-development/engineering/infrastructure/jenkins.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Jenkins
```
Comment (cpanato) [inline operations/research-and-development/engineering/infrastructure/jenkins.md]: this we can wipe out

File: operations/research-and-development/engineering/infrastructure/plugins.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Add plugin
```
Comment (cpanato) [inline operations/research-and-development/engineering/infrastructure/plugins.md]: this we can wipe out

**Review by cpanato — COMMENTED** [review]
some comments

File: operations/research-and-development/engineering/infrastructure/plugins.md (hunk 3 above)
Comment (justinegeffen) [inline operations/research-and-development/engineering/infrastructure/plugins.md]:
```suggestion
```

File: operations/research-and-development/engineering/infrastructure/aws.md (hunk 1 above)
Comment (justinegeffen) [inline operations/research-and-development/engineering/infrastructure/aws.md]:
```suggestion
```

File: operations/research-and-development/engineering/infrastructure/jenkins.md (hunk 2 above)
Comment (justinegeffen) [inline operations/research-and-development/engineering/infrastructure/jenkins.md]:
```suggestion
```

File: operations/research-and-development/engineering/infrastructure/kubernetes-troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Kubernetes Maintenance"
```
Comment (justinegeffen) [inline operations/research-and-development/engineering/infrastructure/kubernetes-troubleshooting.md]:
```suggestion
# Kubernetes Maintenance
```

**Review by justinegeffen — COMMENTED** [review]
Some corrections

**Pull request closed without merging**